StartHere.txt

```diff
@@ -10,7 +10,7 @@ Before you dive deep into this module it'd be best if you made sure that you hav
 *	You know what MVC is about and how ASP.NET MVC works. It's important that you understand how ASP.NET MVC works or 
 	generally what MVC is about. If you are not familiar with the topic take a look at the tutorials at http://www.asp.net/mvc.
 	Or for a funnier start, watch this video: http://channel9.msdn.com/Blogs/matthijs/ASPNET-MVC-2-Basics-Introduction-by-Scott-Hanselman

```

Comment:
I cannot find this video anywhere on the internet now. :( So, you could remove this line.

Reply:
Maybe this? Same presenter and similar topic :)
https://learn.microsoft.com/en-us/shows/asp-net-site-videos/5-minute-introduction-to-aspnet-mvc

Reply:
Nice, this will do it! :)